config/sidekiq.yml

```diff
@@ -6,3 +6,5 @@
 schedule:
   CreditInsufficientNotificationJob:
     cron: '0 0 10 * * 4'  # At 10:00:00 on Thursday => https://crontab.guru/#0_10_*_*_4
+  PaymentPollJob:
+    cron: '*/10 * * * *' # Every 10 minutes => https://crontab.guru/#*/10_*_*_*_*
```

Comment:
I think the frequency could be higher. The load isn't too much and for the use-case of this job 10 minutes is a too long wait for the user.

Reply:
This is a fallback only, by default the user would be redirected back to the system and the payment would be processed instant. However, when this fails (for example the user forgets to click on the 'go back to shop' button) this poll job will make the payment done in the system. Since it is a fallback only I think 10 minutes is okay, or do you think it should be more often even if it's a fallback only?

Reply:
Yes I think that is the case: more often even for the fallback.

When does this happen? I think most often if the callback fails e.g. the user closes the browser after they paid in their banking app or they lose internet connection...
I'd be ok with waiting 5 minutes to see the payment complete, but not with 10. 

What happens when they don't see the payment showing up within a reasonable amount of time? I assume they contact the treasurer

Brainfart: we could also introduce a little more logic to only check recent payments (within the last 30 minutes) with short intervals, and check **all** payment only on long intervals. But that's a lot of code for a minor thing.